### PR TITLE
add more colour information for pngs

### PR DESCRIPTION
**Pull request by Reettaphant**
Add more colour model information for pngs from metadata extractor.

### CODE REVIEW

File: image-loader/app/lib/imaging/FileMetadataReader.scala [hunk 1]
```diff
@@ -125,8 +125,13 @@ object FileMetadataReader {
 
       val pngDir = metadata.getFirstDirectoryOfType(classOf[PngDirectory])
 
-      Map("colorType" -> pngDir.getDescription(PngDirectory.TAG_COLOR_TYPE),
-        "bits" -> pngDir.getString(PngDirectory.TAG_BITS_PER_SAMPLE))
+      Map(
+        "colorType" -> Option(pngDir.getDescription(PngDirectory.TAG_COLOR_TYPE)),
+        "bits" -> Option(pngDir.getDescription(PngDirectory.TAG_BITS_PER_SAMPLE)),
```
Comment (paperboyo) [inline image-loader/app/lib/imaging/FileMetadataReader.scala]: Maybe safer to call it `bitsPerSample` if only for consistency?

File: image-loader/app/lib/imaging/FileMetadataReader.scala [hunk 2]
```diff
@@ -125,8 +125,13 @@ object FileMetadataReader {
...
+        "paletteHasTransparency" -> Option(pngDir.getDescription(PngDirectory.TAG_PALETTE_HAS_TRANSPARENCY)),
```
Comment (kenoir) [inline image-loader/app/lib/imaging/FileMetadataReader.scala]: Should this end up being a boolean?

Comment (Reettaphant) [inline image-loader/app/lib/imaging/FileMetadataReader.scala]: The metadata extractor seems to store this as a yes/no string

**Comment by kenoir** [comment]
:+1: apart from @paperboyo's  comment.

**Pull request merged**